go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/siderolabs/gen
 
-go 1.21.0 // Starting with Go 1.21 you have to provide the third digit too.
+go 1.22.0 // Starting with Go 1.21 you have to provide the third digit too.
```

Comment:
1.22.1 :sweat_smile:

Reply:
This actually doesn't make a lot of sense since there is nothing new in `go 1.22.1`. It will bump all our projects `go.mod` to  `go 1.22.1` so I would rather not do that.